File: python/model_loader.py
import xgboost as xgb
import pandas as pd


loadpath = 'python/synthetic/xgboost_paras/pawelczyk.model'

bst = xgb.Booster()
bst.load_model(loadpath)

df_train = pd.read_csv('python/synthetic/data/cassini.csv')
df_interest = pd.read_csv('python/synthetic/x_interests/cassini.csv')

X_interest, y_interest = df_interest.drop('y', axis=1), df_interest['y']
d_interest = xgb.DMatrix(X_interest, y_interest)

y_proba_test = bst.predict(d_interest)

File: python/model_loader.R
library(xgboost)

df = read.csv('python/synthetic/data/bn_1.csv')
model <- xgboost::xgb.load('python/synthetic/xgboost_paras/bn_1.model')

X_train <- df[,1:4]
y_train <- df[,5]

dtrain <- xgboost::xgb.DMatrix(
    data = as.matrix(X_train),
    label = y_train
)

y_proba_train <- predict(model, dtrain)